### PR TITLE
feat: do not fail to re-login if previous token is wrong

### PR DESCRIPTION
**Pull request by realSpok**
## Definition of Done
you can test with the following token:
```
eJraWQiOiJvd1ZNUExBckJtcXlSRGh1WGlNaXUwQXFpTW8tVE9EZHRQcUFkU0lxSEw0IiwiYWxnIjoiUlMyNTYifQ.eyJkYXRhIjp7ImRhdGEiOnsidHlwZSI6InVzZXJzIiwiaWQiOiIyODQiLCJhdHRyaWJ1dGVzIjp7ImZpcnN0X25hbWUiOiJEZW1vIiwibGFzdF9uYW1lIjoiVXNlciIsImVtYWlsIjoiaGVsbG9AZm9yZXN0YWRtaW4uY29tIn19fSwiaXNBcHBsaWNhdGlvblRva2VuIjp0cnVlLCJhdWQiOiJGT1JFU1RfVVNFUlMiLCJpc3MiOiJGT1JFU1RfQVVUSEVOVElDQVRJT05fU1lTVEVNIiwiaWF0IjoxNzExNDY3NTQ2LCJleHAiOjQ4NjcyMjc1NDZ9.Vs0nJuPC75u-Mu0VX96-RgK4i2tf8gjTYDC61gtfe3_VQch_R1IieFrVQStaKDyHsZTK8Iy0WbVS7qTdX6aZYyJO_DZurOvYUc2Rcj36oS8WzPYWQFwhIw9fVXxkDGOGbNo5iIw3ZVvgzdiTnEamC0AN7chwAfhJjz--Ulx9CbOXu1UTcdcE2FMbjGVjS1AZh5Il7_YVOpWcMjLu5GXQWBfwsNURY5sDGUIojHRMYlQNkSNb1P7faHjGjApPXfMK9IpnQ1lmmOJXltPgpx2FQbG7KhSmyoX16SP0453Teoaek84eD9aLylwaLrgUk3q746aX37E-Gi_ax6akEZyRVQ
```

### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: test/services/api.unit.test.js [hunk 1]
```diff
@@ -112,5 +112,20 @@ describe('services > API', () => {
       expect(superagent.set).toHaveBeenCalledWith('Authorization', 'Bearer THE-TOKEN');
       expect(superagent.send).toHaveBeenCalledWith();
     });
+    // eslint-disable-next-line jest/prefer-expect-assertions
```
Comment (Scra3) [inline test/services/api.unit.test.js:115]: do we need this line?

Comment (realSpok) [reply test/services/api.unit.test.js:115]: This is debatable:
I dont like specifiying `expect.assertions(7);`, but this is a linting rule on the repo so indeed I need this comment

Comment (Scra3) [reply test/services/api.unit.test.js:115]: We should follow the repository conventions I think :/

**Review by Scra3 — APPROVED** [review]

File: test/services/api.unit.test.js (hunk 1 above)
Comment (realSpok) [reply test/services/api.unit.test.js]: Done

**Pull request merged**